src/content/docs/guide/sync.mdx

```diff
@@ -77,3 +79,10 @@ Or, if you see missing data, force a full sync with:
 atuin sync -f
 ```
   
+## Using a self hosted server
```

Comment:
I think this would probably make more sense in the self hosting section! But otherwise looks great, thank you 🙏

Reply:
I've updated it by renaming 'Overview' under 'Self hosted' to 'Server Setup' and created a new chapter called 'Usage' containing the client setup information.